packages/react/src/routes.tsx

```diff
@@ -0,0 +1,45 @@
+import { Outlet, PathPattern, Route, Routes } from "react-router-dom"
+
+import { BrowserRouter } from "react-router-dom"
+import { ProjectProvider } from "./contexts/Project/ProjectProvider"
+import ProjectTable from "./components/ProjectTable"
+import { FallbackPage } from "./components/FallbackPage"
+import { EntityView } from "./components/EntityView"
+
+export type AppRoutesProps = {
+    CommandCenter: React.ComponentType;
+}
+
+export const AppRoutes = ({
+    CommandCenter
+}: AppRoutesProps) => {
+    return (
+        <BrowserRouter>
+            <Routes>
+                <Route element={<>
+                    <ProjectProvider>
+                        <CommandCenter />
+                        <Outlet />
+                    </ProjectProvider>
```

Comment:
Wrapped the entire app in `ProjectProviderContext`

This was useful for accessing project context from within CommandCenter. This also renders CommandCenter on all routes within the app